docs/stable/dev/building/windows.md

```diff
@@ -58,13 +58,13 @@ go build
 collect2.exe: error: ld returned 5 exit status
 ```
 
-GitHub user [vdmitriyev](https://github.com/vdmitriyev) shared instructions for [building the DuckDB Go client on Windows](https://github.com/marcboeker/go-duckdb/issues/4#issuecomment-2176409066):
+GitHub user [vdmitriyev](https://github.com/vdmitriyev) shared instructions for [building the DuckDB Go client on Windows](https://github.com/duckdb/duckdb-go/issues/4#issuecomment-2176409066):
```

Comment:
Same here, likely needs to keep linking to original repository.

Reply:
fixed